### PR TITLE
Use flake8

### PR DESCRIPTION
**Pull request by meanmail**
Task: [#HSPC-](https://vyahhi.myjetbrains.com/youtrack/issue/HSPC-)

**Dependencies**
- !

**Reviewers**
- [ ] @

**Description**

### CODE REVIEW

File: tests/outcomes/separate_package/find_module_no_info_analyze_imports/dir/main.py [hunk 1]
```diff
@@ -1,2 +1 @@
-import main2
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_imports/dir/main.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_imports/dir/main2.py [hunk 2]
```diff
@@ -1,2 +1 @@
-import main3, main4
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_imports/dir/main2.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_imports_2/dir/main2.py [hunk 3]
```diff
@@ -1,3 +1 @@
-import main3
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_imports_2/dir/main2.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_imports_3/dir/main2.py [hunk 4]
```diff
@@ -1,2 +1 @@
-import main3
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_imports_3/dir/main2.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_name_main/dir/main2.py [hunk 5]
```diff
@@ -1,2 +1 @@
-import main3
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_name_main/dir/main2.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_name_main_2/dir/main2.py [hunk 6]
```diff
@@ -1,2 +1 @@
-import main3
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_name_main_2/dir/main2.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_name_main_3/dir/main2.py [hunk 7]
```diff
@@ -1,2 +1 @@
-import main3
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_name_main_3/dir/main2.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_name_main_4/dir/main2.py [hunk 8]
```diff
@@ -1,2 +1 @@
-import main3
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_name_main_4/dir/main2.py:1]: Same

File: tests/outcomes/separate_package/find_module_no_info_analyze_name_main_4/dir/main3.py [hunk 9]
```diff
@@ -1,2 +1 @@
-import main4
```
Comment (aaaaaa2493) [inline tests/outcomes/separate_package/find_module_no_info_analyze_name_main_4/dir/main3.py:1]: Same

**Review by aaaaaa2493 — CHANGES_REQUESTED** [review]

**Review by inuur — APPROVED** [review]

**Pull request merged**